src/app/multicurrency-landing/multicurrency-landing.component.html

```diff
@@ -0,0 +1,64 @@
+<main>
+  <!-- <app-ticker [campaign]="campaign" [fund]="fund"></app-ticker> -->
+  <div class="hero">
+
+  </div>
+
+  <section class="cta">
+    <div class="b-container">
+      <h1>Vaccines for everyone, everywhere</h1>
+      <div class="multicurrency-location-pick-wrapper" style="padding: 0 0 150px 0;">
```

Comment:
I would avoid inline `style`, especially when mixed with SCSS with a selector matching the same object.

Reply:
I've added the inline css here to override the default styling for the `multicurrency-location-pick-wrapper.` Because it appears in two places, I have the default padding style defined in scss, but for the first occurrence of `multicurrency-location-pick-wrapper` it needs some extra padding, so I've overridden like this instead of making a second, conflicting class name.

Reply:
@alihejazi this sounds like it should just be two different classes